app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Users
+  class RegistrationsController < ApplicationController
+    before_action :ensure_tribunal_case
```

Comment:
I'm not sure if this is always appropriate. I think a user can click "resume a saved case", go to the login screen, and then click "create an account"  So, they could get here *without*  tribunal case existing.